extensions/amp-story/1.0/test/test-analytics.js

```diff
@@ -61,17 +61,27 @@ describes.fakeWin('amp-story analytics', {}, env => {
 
   it('should not mark an event as repeated the first time it fires', () => {
     const trigger = sandbox.spy(analytics, 'triggerEvent');
+    const details = sandbox.spy(analytics, 'updateDetails');
 
     storeService.dispatch(Action.CHANGE_PAGE, {
       id: 'test-page',
       index: 1,
     });
 
     expect(trigger).to.have.been.calledOnceWith('story-page-visible');
-    expect(analytics.updateDetails().pageDetails).to.deep.include({
-      'story-page-visible': {
-        'repeated': false,
-      },
+
+    expect(details).to.have.returned({
```

Comment:
Just realized this might not be desirable to maintain. Thinking of a better way to test this.